src/libraries/System.Net.Http.Json/ref/System.Net.Http.Json.cs

```diff
@@ -8,107 +8,128 @@ namespace System.Net.Http.Json
 {
     public static partial class HttpClientJsonExtensions
     {
-        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("JSON serialization and deserialization might require types that cannot be statically analyzed. Use the overload that takes a JsonTypeInfo or JsonSerializerContext, or make sure all of the required types are preserved.")]
```

Comment:
Could you minimize the diff in this file to just the new API additions, even if they need to be handwritten?

Reply:
I'll update that this AM, thanks for calling attention to that. Question, do you know, does the `ref` generator create the APIs in a consistent way with a deterministic order?

Reply:
If this got rearranged by the tool we should keep the changes since it would contribute to less noise in future runs.

Reply:
I can leave it as is then, since this was a result of the tool.